src/Block/MediaBlockService.php

```diff
@@ -193,8 +195,15 @@ private function getFormatChoices(?MediaInterface $media = null): array
         return $formatChoices;
     }
 
-    private function getMediaBuilder(): FormBuilderInterface
+    /**
+     * @param AdminFormMapper<object> $form
+     */
+    private function getMediaBuilder(AdminFormMapper $form): FormBuilderInterface
     {
+        if (!$form instanceof AdminFormMapper) {
+            throw new \InvalidArgumentException('Media Block requires to be used in the Admin context');
+        }
```

Comment:
This check must be in the `configureEditForm` method, (before the getMediaBuilder call)

Reply:
Ok like that so ? :)

Reply:
yes